src/cpp/rtps/builtin/liveliness/WLP.h

```diff
@@ -303,6 +304,8 @@ class WLP
 #if HAVE_SECURITY
     std::shared_ptr<ITopicPayloadPool> secure_payload_pool_;
 #endif // if HAVE_SECURITY
+
+    static constexpr uint32_t builtin_participant_data_max_size = 100;
```

Comment:
do we really need it to be defined as static?

Reply:
If it isn't, then you cannot access it without a class instance, and since it is a constant and it came from a `#define` I think this is more suitable.